helm-chart/values.yaml

```diff
@@ -364,56 +355,30 @@ grafana:
       enabled: false
 
 ingress:
+  ## @param ingress.enabled Enable ingress record generation for CVAT
+  ##
   enabled: false
-
-# In case you need an ingress, write them manually in your my.values.eml, see example below:
-#  enabled: true
-#  annotations:
-#    traefik.ingress.kubernetes.io/router.entrypoints: web
-#    kubernetes.io/ingress.class: traefik
-#  hosts:
-#    - host: cvat.local
-#      paths:
-#        - path: /api
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /admin
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /static
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /django-rq
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /profiler
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path : /
-#          pathType: "Prefix"
-#          service:
-#            name: frontend-service
-#            port: 80
-#  tls:
-#    - hosts:
-#      - <your_domain>
-#    secretName: ingress-tls-cvat
+  ## @param ingress.pathType Ingress path type
+  ##
+  pathType: Prefix
```

Comment:
Why do we need this to be configurable? We know that our paths are prefixes, no other option would make sense.